EIPS/eip-3754.md

```diff
@@ -0,0 +1,53 @@
+---
+eip: 3754
+title: A Vanilla Non-Fungible Token Standard
+description: A vanilla NFT standard by removing data fields and functions related to `URI`.
+author: Simon Tian <simon@dtopia.me>
+discussions-to: https://github.com/ethereum/EIPs/issues/3753
+status: Draft
+type: Standards Track,
+category (*only required for Standards Track): ERC
+created: 2021-08-21
+requires (*optional): <EIP number(s)>
+replaces (*optional): <EIP number(s)>
+---
+
+## Abstract
+The ERC721 NFT standard has gained much popularity for its capability of being associated with external contents via the data field `URI`. However, this is also handicapping NFTs not having external associations. This standard is going to simplifiy ERC721 by removing data fields and functions related to `URI` and introduce a vanilla NFT that is ideal for representing abstract ownership such as rights. This can open up more possibilities by allowing layers of abstraction being built on top of it.
```

Comment:
```suggestion
The [ERC-721](./eip-721.md) NFT standard has gained much popularity for its capability of being associated with external contents via the data field `URI`. However, this is also handicapping NFTs not having external associations. This standard is going to simplifiy ERC-721 by removing data fields and functions related to `URI` and introduce a vanilla NFT that is ideal for representing abstract ownership such as rights. This can open up more possibilities by allowing layers of abstraction being built on top of it.
```